_config.yml

```diff
@@ -0,0 +1,4 @@
+theme: jekyll-theme-primer
```

Comment:
So is `jekyll-theme-primer` the style used by this website?
Another theme seems to be defined in the Gemfile:
```shell
# This is the default theme for new Jekyll sites. You may change this to anything you like.
gem "minima", "~> 2.0"
```